src/tiledimage.js

```diff
@@ -1965,6 +1965,7 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
             imageZoom > this.smoothTileEdgesMinZoom &&
             !this.iOSDevice &&
             this.getRotation(true) % 360 === 0 && // TODO: support tile edge smoothing with tiled image rotation.
+            this._drawer.viewer.viewport.getFlip() === false && // TODO: support tile edge smoothing with tiled image flip.
```

Comment:
Thank you for adding the comment. One detail: It's not the tiledImage flip, but the viewport flip. Please update the comment to make that clear. 

It made me wonder why we were only doing this for the tiledImage rotation and not the viewport rotation (and likewise wonder whether we should be doing it for the tiledImage flip). My testing shows it's only necessary for those two; I'm not sure why. At any rate, it might be worth adding to these comments to say that viewport rotation is not a problem, and tiledImage flip is not a problem.

Reply:
sorry for the mistake and thanks for clarification :)

done in f004ecf

Do you want me to add your last  point to those two comments ?

Reply:
Thank you, and yes, if you wouldn't mind!

Reply:
no problem, here it is f96db3c !